pyproject.toml

```diff
@@ -102,7 +108,12 @@ testpaths = ["test/python"]
 addopts = ["-ra", "--strict-markers", "--strict-config", "--showlocals"]
 log_cli_level = "INFO"
 xfail_strict = true
-filterwarnings = ["error"]
+filterwarnings = ["error",
+    'ignore::ImportWarning',
+    'ignore::PendingDeprecationWarning',
+]
```

Comment:
Please be as specific as possible with these filters. As of now, you are filtering _every_ import and pending deprecation warning which is a no-go.
Most of these are important and should be addressed.
The one import warnings you showed me privately can be excluded here.
What are the pending deprecation warnings you are getting?

Reply:
I am getting some `PendingDeprecationWarning` as the AerSimulator is replacing the current QasmSimulator. I tested the script with the AerSimulator and there I have the issue that I can not use the stabilizer Simulator backend which critically limits the scalability of the simulation. Therefore I would like to keep it as it is right now and instead open an issue to switch out the simulator.

Reply:
Please open such an issue and make sure that the warning filters are as tight as possible.

Reply:
I added the issue https://github.com/cda-tum/qecc/issues/66 and updated the filter.